website/docs/r/lb_server_pool.html.markdown

```diff
@@ -0,0 +1,129 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_lb_server_pool"
+sidebar_current: "docs-vcd-resource-lb-server-pool"
+description: |-
+  Provides a NSX load balancer server pool data source.
```

Comment:
Description is incorrect (copy/paste from data source).

Reply:
Sorry. Indeed I did some copy paste and failed here.

Reply:
Thanks, but now in a different place of `website/docs/r/lb_server_pool.html.markdown` you're saying:
> Provides a vCloud Director Edge Gateway Load Balancer Server Pool **data source**.
:)